### PR TITLE
Absolute path quick-fix in QuackRepl.php

### PR DESCRIPTION
**Pull request by joaolucasl**
Added BASE_PATH to the QuackToolkit import in QuackRepl.php. The previous version wouldn't work in one of my PCs and this fix makes it clear where the file is.

### CODE REVIEW

**Comment by joaolucasl** [comment]
'self' LGTM

File: src/repl/QuackRepl.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
  * along with Quack.  If not, see <http://www.gnu.org/licenses/>.
  */
 define('BASE_PATH', __DIR__ . '/..');
-require_once '../toolkit/QuackToolkit.php';
+require_once(BASE_PATH.'/toolkit/QuackToolkit.php');
```
Comment (haskellcamargo) [inline src/repl/QuackRepl.php]: Well, the only issue is that our pattern for operators is:
 `<expr> <op> <expr>`, which is space separated: `BASE_PATH . '/toolkit...`.

Comment (joaolucasl) [inline src/repl/QuackRepl.php]: Updated :)

**Comment by haskellcamargo** [comment]
LGTM.

**Pull request merged**